File: AntiClown.Entertainment.Api.Dto/F1Predictions/ChampionshipPredictions/F1ChampionshipResultsDto.cs
namespace AntiClown.Entertainment.Api.Dto.F1Predictions.ChampionshipPredictions;

public record F1ChampionshipResultsDto
{
    public bool HasData { get; set; }
    public F1ChampionshipPredictionTypeDto Stage { get; set; }
    public bool IsOpen { get; set; }
    public string[] Standings { get; set; }
}

File: AntiClown.Web.Front/src/Dto/F1Predictions/F1ChampionshipResultsDto.ts
import { F1ChampionshipPredictionTypeDto } from "./F1ChampionshipPredictionTypeDto";

export interface F1ChampionshipResultsDto {
  hasData: boolean;
  stage: F1ChampionshipPredictionTypeDto;
  isOpen: boolean;
  standings: string[];
}
